massifs/watcher/logtails_test.go

```diff
@@ -3,12 +3,74 @@ package watcher
 import (
 	"fmt"
 	"reflect"
+	"slices"
 	"testing"
 
 	"github.com/Azure/azure-sdk-for-go/sdk/storage/azblob"
 	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
 )
 
+func Test_MaxSuffled(t *testing.T) {
```

Comment:
I don't know what Test_MaxSuffled means? is it meant to be MaxShuffled? Can we have a test description as well please, as it doesn't seem to be testing that the tenants are shuffled, but rather checking that the returned list of tenants in the massif + seal collator is correct.